cursive-core/src/printer.rs

```diff
@@ -46,8 +48,7 @@ pub struct Printer<'a, 'b> {
     /// Offset into the view for this printer.
     ///
     /// The view being drawn can ignore this, but anything to the top-left of
-    /// this will actually be ignored, so it can be used to skip this part.
-    ///
+    /// this will actually be ignored, so it can be used to skip this part.\
```

Comment:
Here too I think the formula for the actual location is independent from "the view can ignore this", so I don't think it belongs in the same paragraph.